.github/workflows/upload-to-pypi.yml

```
@@ -0,0 +1,94 @@
+name: Upload-ansible-tarball-pypi
+on:
+  workflow_dispatch:
+    inputs:
+      ansible_version:
+        description: 'Release Version. Example : 11.1.0'
+        required: true
+      ansible_major_version:
+        description: 'Exmaple 11'
+        required: true
+
+env:
+  BUILD_DATA_DIR: "antsibull/build/ansible-build-data/${{ inputs.ansible_major_version }}"
+  BUILDFILE: "ansible-${{ inputs.ansible_major_version }}.build"
+  DEPSFILE: "ansible-${{ inputs.ansible_version }}.deps"
+
+jobs:
+  publish:
+    name: 'Upload Ansible community distribution (${{ inputs.ansible_version }}) to PyPI'
+    runs-on: ubuntu-latest
+    environment:
+      name: release
```

use a more accurate name
```suggestion
      name: testpypi
```